plugins/modules/aci_l3out_floating_svi_path_secondary_ip.py

```diff
@@ -0,0 +1,398 @@
+---
+module: aci_l3out_floating_svi_path_secondary_ip
+short_description: Manages Layer 3 Outside (L3Out) Floating SVI Path Attribute's Secondary IP addresses(l3ext:Ip)
+description:
+- Manages L3Out Floating SVI path attribute's secondary IP addresses on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+    required: true
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+    required: true
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+    required: true
+  interface_profile:
+    description:
+    - Name of the interface profile.
+    type: str
+    aliases: [ interface_profile_name, logical_interface ]
+    required: true
+  pod_id:
+    description:
+    - Pod to build the interface on.
+    type: str
+    required: true
+  node_id:
+    description:
+    - Node to build the interface on for Port-channels and single ports.
+    type: str
+    required: true
+  encap:
+    description:
+    - Encapsulation on the interface (e.g. "vlan-500")
+    type: str
+    required: true
+  domain:
+    description:
+    - This option allows virtual machines to send frames with a mac address.
+    type: str
+    required: true
+  domain_type:
+    description:
+    - The domain type of the path.
+    type: str
+    choices: [ physical, vmware ]
+    required: true
+  secondary_ip:
+    description:
+    - The secondary floating IP address.
+    type: str
+    aliases: [ secondary_floating_address ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+
+notes:
+- The C(tenant), C(l3out), C(logical_node_profile), C(logical_interface_profile) and C(floating_svi) must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant), M(cisco.aci.aci_l3out), M(cisco.aci.aci_l3out_logical_node_profile), M(cisco.aci.aci_l3out_logical_interface_profile), \
+  M(cisco.aci.aci_l3out_floating_svi) and M(cisco.aci.aci_l3out_floating_svi_path) can be used for this.
+seealso:
+- module: cisco.aci.aci_tenant
+- module: cisco.aci.aci_l3out
+- module: cisco.aci.aci_l3out_logical_node_profile
+- module: cisco.aci.aci_l3out_logical_interface_profile
+- module: cisco.aci.aci_l3out_floating_svi
+- module: cisco.aci.aci_l3out_floating_svi_path
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(l3ext:Ip)
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Shreyas Srish (@shrsr)
+"""
+
+EXAMPLES = r"""
+- name: Create a Floating SVI path attribute secondary IP
+  cisco.aci.aci_l3out_floating_svi_path:
```

Comment:
```suggestion
  cisco.aci.aci_l3out_floating_svi_path_secondary_ip:
```
Also update other examples.

Reply:
Good catch.
thanks!